srcpkgs/prometheus/template

```diff
@@ -1,8 +1,11 @@
 # Template file for 'prometheus'
 pkgname=prometheus
-version=2.53.1
-revision=2
+version=3.0.1
+revision=1
 build_style=go
+if [ "${XBPS_BUILD_ENVIRONMENT}" != "void-packages-ci" ]; then
+  make_check_args+="-skip (TestQueryLog|TestDelayedCompaction)"
+fi
```

Comment:
This block should be after `make_dirs=...`. It also needs a space at the beginning of the string, that's my bad for not suggesting it correctly:
```
make_check_args+=" -skip (TestQueryLog|TestDelayedCompaction)"
```

Reply:
No problem, fixed.